x/erc20/types/proposal.go

```diff
@@ -114,7 +114,7 @@ func validateIBC(metadata banktypes.Metadata) error {
 func ValidateErc20Denom(denom string) error {
 	denomSplit := strings.SplitN(denom, "/", 2)
 
-	if len(denomSplit) != 2 || denomSplit[0] != ModuleName {
+	if denomSplit[0] != ModuleName {
 		return fmt.Errorf("invalid denom. %s denomination should be prefixed with the format 'erc20/", denom)
 	}
 
```

Comment:
Calrification: So this means all erc20 denominated tokens carry prefix like "evmos/" ?

Reply:
@Prosp3r not quite, all erc20 tokens that are deployed by the erc20 module have the `erc20` prefix. So if you register a coin, say `ibc/12345...` then the deployed erc20 token would looks smth like `erc20/12345...`